### PR TITLE
Update "software" method in Entertainment

### PR DESCRIPTION
**Pull request by phallobst**
See #4410

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
       - hardware
```
Comment (Carlgo11) [inline _data/entertainment.yml]:
```suggestion
      - u2f
```

File: _data/entertainment.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
```
Comment (Carlgo11) [inline _data/entertainment.yml:52]:
```suggestion
      - totp
      - proprietary
```

File: _data/entertainment.yml [hunk 3]
```diff
@@ -145,8 +145,8 @@ websites:
     url: https://www.smashcast.tv/
     img: smashcast.png
     tfa:
-      - software
-    doc: https://help.smashcast.tv/customer/en/portal/articles/2779887
+      - totp
+    doc: https://support.desk.com/customer/portal/articles/1894034
```
Comment (Carlgo11) [inline _data/entertainment.yml]: That doc link is for Salesforce Desk. Looks like Smashcast's help site is down at the moment so let's leave it empty for now.
```suggestion
```

File: _data/entertainment.yml [hunk 4]
```diff
@@ -201,6 +201,6 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
```
Comment (Carlgo11) [inline _data/entertainment.yml:204]:
```suggestion
      - totp
      - proprietary
```

File: _data/entertainment.yml [hunk 5]
```diff
@@ -201,6 +201,6 @@ websites:
...
       - hardware
```
Comment (Carlgo11) [inline _data/entertainment.yml]:
```suggestion
      - u2f
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**